src/frontend/web_application/src/components/ContactDetails/components/OrgaForm/index.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../../../Icon';
+import Button from '../../../Button';
+import { FieldErrors, Fieldset, Legend, TextFieldGroup, FormGrid, FormRow, FormColumn } from '../../../form';
+
+import './style.scss';
+
+
+class OrgaForm extends Component {
+  static propTypes = {
+    errors: PropTypes.arrayOf(PropTypes.string),
+    onSubmit: PropTypes.func.isRequired,
+    __: PropTypes.func.isRequired,
```

Comment:
As a user I think, I'll be frustrated if I have to delete then re-add a whole organization just because of a typo to fix.